src/main/components/DataTree.jsx

```diff
@@ -9,14 +10,20 @@ const DataTree = (props) => {
     const tree = JsonTree.create(data, htmlElement);
   };
 
+  const changeObject = (val) => {
+    console.log(val);
+  };
+
   useEffect(() => {
     const test = document.getElementById(`tree-${treeCount}`);
-    renderTree(test);
+    // renderTree(test);
```

Comment:
Spoke to Don. This comment was left until the team decides on the outcome of the new imported library